services/github/github-api-provider.js

```diff
@@ -63,13 +93,7 @@ class GithubApiProvider {
   }
 
   tokenForUrl(url) {
-    const { globalToken } = this
-    if (globalToken) {
```

Comment:
~~Does this change mean that we'll always use a token from the pool here, even if the global token was set?~~

I see this check still exists but has shifted down below, scratch this!